src/main/java/config/AppConfing.java

```diff
@@ -4,24 +4,31 @@
 import javax.sql.DataSource;
 import models.User;
 import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
 import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
 
 @Configuration
+@PropertySource("classpath:db.properties")
 @ComponentScan(basePackages = {
         "service",
         "dao"
 })
 public class AppConfing {
+    @Autowired
+    private Environment environment;
```

Comment:
We can initialise this field through the constructor as prescribed in the checklist